src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/UserAccessServiceTest.kt

```diff
@@ -647,6 +647,92 @@ class UserAccessServiceTest {
     assertThat(userAccessService.currentUserCanManagePremisesLostBeds(temporaryAccommodationPremisesNotInUserRegion)).isFalse
   }
 
+  @ParameterizedTest
+  @EnumSource(value = UserRole::class, names = [ "CAS1_MANAGER", "CAS1_MATCHER" ])
```

Comment:
I think perhaps we should be adding in `CAS1_FUTURE_MANAGER` from the get-go? As that is the role which will ultimately have sole access for managing lost beds on the new `/cas1/premises/{premisesId}/out-of-service-beds` paths

Reply:
Good catch, I've changed the tests that use these roles to include `CAS1_FUTURE_MANAGER`.